File: game-admin/src/main/java/com/dkm/goods/services/IGoodService.java
package com.dkm.goods.services;

import com.dkm.goods.entities.vo.GoodsVo;
import com.dkm.utils.bean.ResultVo;
import java.util.List;
import kotlin.Metadata;
import org.jetbrains.annotations.NotNull;


public interface IGoodService {
   @NotNull
   List<GoodsVo> getAllGoods();

   @NotNull
   ResultVo addGoodsItem(@NotNull GoodsVo var1);

   @NotNull
   ResultVo updateItemById(long var1, @NotNull GoodsVo var3);

   @NotNull
   ResultVo deleteItemById(@NotNull String var1);
}

File: game-admin/src/main/java/com/dkm/goods/services/IGoodService.kt
package com.dkm.goods.services


import com.dkm.goods.entities.vo.GoodsVo
import com.dkm.utils.bean.ResultVo

interface IGoodService {
    fun getAllGoods(): List<GoodsVo>
    fun addGoodsItem(goods: GoodsVo): ResultVo
    fun updateItemById(id: Long, goods: GoodsVo): ResultVo
    fun deleteItemById(id: String): ResultVo
}